src/modules/data/blocks/ticket/__tests__/sagas.test.js

```diff
@@ -395,6 +407,41 @@ describe( 'Ticket Block sagas', () => {
 		} );
 	} );
 
+	describe( 'removeTicketsBlock', () => {
+		it( 'should remove tickets block', () => {
+			const gen = cloneableGenerator( sagas.removeTicketsBlock )();
+			expect( gen.next().value ).toEqual(
+				select( selectors.getSharedTicketsCount )
+			);
+
+			const clone1 = gen.clone();
+			expect( clone1.next( 2 ).done ).toEqual( true );
+
+			const clone2 = gen.clone();
+			expect( JSON.stringify( clone2.next( 0 ).value ) ).toEqual(
```

Comment:
I'd just do a snapshot here @paulmskim. It's a lot of work in some cases to model the state correct, and make sure separate calls are using the same functions.